packages/manager/apps/catalog-revamp/src/utils/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { groupBy, includes, isEmpty, map, uniq } from 'lodash';
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+type ProductKey = 'universe' | 'category' | 'name';
+
+export const groupItems = (itemsToGroup: Product[]) => {
+  return groupBy(itemsToGroup, 'universe');
+};
+
+export const filterByEnum = (
+  product: Product,
+  query: any[],
```

Comment:
Can you be more precise with the type of query ?

Reply:
Done